bot/main.py

```diff
@@ -210,25 +244,43 @@ async def backend_info(ctx):
     )
 
 
-def is_academic(ctx: commands.Context):
-    """Checks if the server is an academic server."""
+@backend_info.error
+async def backend_info_error(ctx, error):
+    """If the author of the message is not a bot admin then reply accordingly."""
+    if (
+        isinstance(error, commands.CheckFailure)
+        and isinstance(error.original, CheckFailedException)
+        and error.original.check_name == "is_author_bot_admin"
```

Comment:
In my testing, this codepath isn't being triggered. So the thing is the `.original` attribute does not contain the original error raised.